index.bs

```diff
@@ -300,15 +300,19 @@ parallel</a>:
 
 <pre class="idl">
 partial interface DocumentOrShadowRoot {
-  readonly attribute HTMLVideoElement? pictureInPictureElement;
+  readonly attribute Element? pictureInPictureElement;
 };
 </pre>
 
 The {{pictureInPictureElement}} attribute's getter must run these steps:
 
 1. If the <a>context object</a> is a <a for=/>shadow root</a> and its
     <a for=DocumentFragment>host</a> is not <a>connected</a>, then return null.
-2. Return the Picture-in-Picture video element.
+2. Let |candidate| be the result of <a>retargeting</a> Picture-in-Picture
```

Comment:
Is "retargeting" defined in the spec?

Reply:
It is. See https://dom.spec.whatwg.org/#retarget